.buildkite/deps-pipeline.yml

```diff
@@ -1,10 +1,10 @@
 steps:
   - label: "build-deps-x86"
     commands:
-      - resources/kernel/make_kernel_image.sh -f elf -k vmlinux-hello-busybox -w /tmp/vmlinux_busybox -j 2
-      - resources/kernel/make_kernel_image.sh -f elf -k vmlinux-hello-busybox -w /tmp/vmlinux_busybox -j 2 -h
-      - resources/kernel/make_kernel_image.sh -f bzimage -k bzimage-hello-busybox -w /tmp/vmlinux_busybox -j 2
-      - resources/kernel/make_kernel_image.sh -f bzimage -k bzimage-hello-busybox -w /tmp/vmlinux_busybox -j 2 -h
+      - resources/kernel/make_kernel_initramfs_image.sh -f elf -k vmlinux-hello-busybox -w /tmp/vmlinux_busybox -j 2
```

Comment:
Is this script busybox-specific? If so, `make_busybox_image` might be a more accurate name/description.

Reply:
Yup, it builds kernels witth busybox initramfs'es. I'll re-rename it.